public/js/post/post.js

```diff
@@ -116,7 +126,8 @@ var MultipleCourseCode = React.createClass({
 var SpecialistPost = React.createClass({
     getInitialState: function() {
         return {
-            selected: true
+            selected: true,
+            activeCourses: activeCourses.slice()
```

Comment:
I'm not sure what the `activeCourses` on the right is. It seems like this would be a good place to call the function which computes which courses are active.